src/register/Register.jsx

```diff
@@ -0,0 +1,272 @@
+import React, { useState } from "react";
+import "./register.css";
+
+
+export default function Register() {
+    // set user data state
```

Comment:
unnecessary comment. Commenting isn't really a good practice. It's a good practice if your code can be readable, that way you never need comments. 
Only comment when you find it's absolutely necessary or these is some complex non obvious non readable code. 
So please remove this

Reply:
will do that 👍